### PR TITLE
Update and private networks

### PR DESCRIPTION
**Pull request by jgtormo**

### CODE REVIEW

File: docs/cudos-intercloud/usage/via-user-interface/private-networks.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Creating a VM in a Private Network
+The process is the same as for any other VM, expcet that you need to select the private network that you want in the dropdown menu of the Networking box.
+Optionally, you can assign a public IP address, so that it can be reached from the outside as well.
+
+Please note that **you will always need at least one machine in your private network with a public IP address**, to act as a jump host.
+You need to connect via SSH to all machines remotely, so you need one that is going to act as the point of access to your network.
+It is recommended that this jump host is dedicated to this purpose, though it's not essential.
+
+## Connecting to a VM in a Private Network
```
Comment (nbaum) [inline docs/cudos-intercloud/usage/via-user-interface/private-networks.md:36]: I would note that this is only one way to use a jump host. It isn't _necessary_ to use the SSH jump host feature, even if it is the most convenient method. You can also SSH into it normally and then SSH again _from_ there. Or even proxy/tunnel some other protocol entirely.

So I might make it clearer that this the "SSH jump host" approach to remote access, which happens to be recommended, but it's just one of the approaches.

File: docs/cudos-intercloud/usage/via-user-interface/private-networks.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+ssh -J root@1.2.3.4 root@10.0.0.8
+``` 
+
+## Billing
+Private networks are billed separately from VMs, but they are also billed hourly.
+The price per network depends on the datacentre in which it's created in, but they are all around $0.0045 per hour.
+Please visit [the network creation page](https://intercloud.cudos.org/#/networks/create) to see the exact prices in each datacentre.
+
+As a side note, IP addresses are billed at around $0.0035 per day in most datacentres.
```
Comment (nbaum) [inline docs/cudos-intercloud/usage/via-user-interface/private-networks.md:52]: *Per hour

File: docs/cudos-intercloud/usage/via-user-interface/private-networks.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+You can find the exact values in the [VM creation page](https://intercloud.cudos.org/#/create).
+Thus, if you are planning on deploying a large number of VMs and your setup allows for most of your VMs to not have a public IP address, this could also be a way to decrease some cost, as well as improve security.
```
Comment (nbaum) [inline docs/cudos-intercloud/usage/via-user-interface/private-networks.md:54]: I would drop the thus.

File: docs/cudos-intercloud/usage/via-user-interface/private-networks.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+CUDOS Intercloud's suppliers all use IPv4 at the moment.
+:::
+
+After that, your network will be created and you will be able to choose it in the `Networking` box of the VM creation page.
+
+## Creating a VM in a Private Network
+The process is the same as for any other VM, expcet that you need to select the private network that you want in the dropdown menu of the Networking box.
+Optionally, you can assign a public IP address, so that it can be reached from the outside as well.
+
+Please note that **you will always need at least one machine in your private network with a public IP address**, to act as a jump host.
```
[the remark this answers is not in the document]
Comment (jgtormo) [reply docs/cudos-intercloud/usage/via-user-interface/private-networks.md]: I think there's no harm mentioning it

**Pull request merged**